src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlSecurityTokenHandler.cs

```diff
@@ -107,7 +108,19 @@ public IEqualityComparer<SamlSubject> SamlSubjectEqualityComparer
         /// <summary>
         /// Gets or set the <see cref="SamlSerializer"/> that will be used to read and write a <see cref="SamlSecurityToken"/>.
         /// </summary>
-        public SamlSerializer Serializer { get; set; } = new SamlSerializer();
+        /// <exception cref="ArgumentNullException">'value' is null.</exception> 
+        public SamlSerializer Serializer
+        {
+            get { return _serializer; }
+
+            set
+            {
+                if (value == null)
```

Comment:
use 'null condition operators' pattern, always use LogHelper to throw exceptions.
_serializer = value ?? LogHelper.LogArgumentNullException(nameof(value))

Reply:
Done.